scripts/mirrorlist

```diff
@@ -0,0 +1,3 @@
+Server = http://mirrors.evowise.com/archlinux/$repo/os/$arch
```

Comment:
While at it, maybe add https://mirrors.edge.kernel.org/archlinux/? It is tier 1 contrary to evowise.com.